### PR TITLE
[Intl] Add phpdoc

### PR DESCRIPTION
**Pull request by ro0NL**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no     
| Deprecations? | no
| Tests pass?   | yes    
| Fixed tickets | https://github.com/symfony/symfony/pull/31325#discussion_r279843099
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

File: src/Symfony/Component/Intl/Tests/TimezonesTest.php [hunk 1]
```diff
@@ -595,8 +604,10 @@ public function testGetCountryCodeWithUnknownTimezone()
      */
     public function testGetGmtOffsetAvailability(string $timezone)
     {
-        $this->assertInternalType('int', Timezones::getRawOffset($timezone));
-        $this->assertInternalType('string', Timezones::getGmtOffset($timezone));
+        Timezones::getRawOffset($timezone);
+        Timezones::getGmtOffset($timezone);
+
+        $this->addToAssertionCount(1);
```
Comment (jakzal) [inline src/Symfony/Component/Intl/Tests/TimezonesTest.php]: This looks weird. What's the point of such test?

File: src/Symfony/Component/Intl/Timezones.php [hunk 2]
```diff
@@ -76,11 +83,17 @@ public static function getGmtOffset(string $timezone, int $timestamp = null, str
...
+    /**
+     * @throws MissingResourceException if the timezone identifier has no associated country code
+     */
     public static function getCountryCode(string $timezone): string
     {
         return self::readEntry(['ZoneToCountry', $timezone], 'meta');
     }
 
+    /**
+     * @throws MissingResourceException if the the country code does not exists
```
Comment (stof) [inline src/Symfony/Component/Intl/Timezones.php]: duplicated `the`

**Review by stof — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Intl/Tests/TimezonesTest.php (hunk 1 above)
Comment (ro0NL) [reply src/Symfony/Component/Intl/Tests/TimezonesTest.php]: it ensures for each timezone we have offset (raw+labeled).

It's ensured by return-type, so if something unexpected happens it violates and this test fails. It's just to notify us if we ever add a timezone with inconsistent/incomplete data.

File: src/Symfony/Component/Intl/Timezones.php (hunk 2 above)
Comment (ro0NL) [reply src/Symfony/Component/Intl/Timezones.php]: fixed.

File: src/Symfony/Component/Intl/Tests/TimezonesTest.php (hunk 1 above)
Comment (fabpot) [reply src/Symfony/Component/Intl/Tests/TimezonesTest.php]: Maybe we can add a comment on each such assertions to explain the goal?

Comment (ro0NL) [reply src/Symfony/Component/Intl/Tests/TimezonesTest.php]: done.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @ro0NL.

**Pull request merged**